migrate-to-github/README.md

```diff
@@ -0,0 +1,77 @@
+Check the [what-to-migrate.md]() file first, please.
+
+### How we want to do it
+
+We will use the XXX tool (not released yet).
+  - still thinking about its name :D I think about two possibilities:
+    - `volagit` or `forgit` -> these are just random names containing `git`... I think
+       about them because they sounds cool :D
+
```

Comment:
There used to be some feature or tool in the Pagure project allowing us to "export" everything out (including issues/PRs?).  We should perform such a dump, and archive it somewhere.  Ideally, each of the artificially created issues or PRs should refer to that archive if anyone wanted to do some archeology ... :)

Honestly, it would be just enough for me to use this template:
```
Issue: <copied topic>

This is copy-pasted issue from http://pagure.io/copr/copr/issue/<ID>.  In case of any problem, feel free to take a look at the archived repo here.
```

Reply:
sounds good, this archive will live somewhere inside fedora-copr namespace? (like some new repo where we will just store this dump)

Reply:
Not sure, I'm not even sure how it works.  I only found the way to regenerate the issues/prs repos -> you need to go to pagure project -> settings -> regenerate repos.  It would be nice to clone them I guess, somehow?

Reply:
@praiskup I tried to regenerate something on my dummy repo but nothing happenned so I am not sure how this works :/